src/components/comment-placer/fade-and-grow-transition/FadeAndGrowTransition.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import AnimateHeight from 'react-animate-height';
+
+export default class FadeAndGrowTransition extends Component {
+    static propTypes = {
+        children: PropTypes.node.isRequired,
+        duration: PropTypes.number,
+        animateOnMount: PropTypes.bool,
+        animateOnUnmount: PropTypes.bool,
+        triggerUnmount: PropTypes.bool,
```

Comment:
I've used `triggerUnmount` for two reasons:
1. I think it's easier to understand which action this prop will trigger;
2. The `in` prop is used only on `FadeTransition` component as it uses `react-transition-group`. `FadeAndGrow` component does not use `react-transition-group`.

But I can change it to `in` on both components.

Reply:
Since both are a Transition component, I would go with `in` because it's a relative standard prop in react that means animateIn when true and animateOut when false.